content/en/tracing/setup/dotnet.md

```diff
@@ -183,15 +185,16 @@ CORECLR_ENABLE_PROFILING=1
 CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
 ```
 
-`CORECLR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and `DD_INTEGRATIONS` is set globally for all processes.
+`CORECLR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and environment variables `DD_INTEGRATIONS` and `DD_DOTNET_TRACER_HOME` set globally for all processes.
 
-If you did not use the MSI installer, set all four environment variables:
+If you did not use the MSI installer, set all five environment variables:
```

Comment:
```suggestion
If you did not use the MSI installer, set the following environment variables:
```

Reply:
As noted above, I'd like to keep this specific number in the documentation to avoid any confusion after reading about the previous two environment variables.